root/static/scripts/edit/URLCleanup.js

```diff
@@ -456,7 +459,13 @@ const CLEANUPS = {
     },
   },
   'amazon': {
-    match: [new RegExp('^(https?://)?(((?!music)[^/])+\.)?(amazon\\.(com|ca|co\\.uk|fr|ae|at|de|it|sg|co\\.jp|jp|cn|es|in|nl|com\\.br|com\\.mx|com\\.au|com\\.tr)|amzn\\.com)', 'i')],
+    match: [new RegExp(
+      '^(https?://)?(((?!music)[^/])+\.)?(amazon\\.' +
+      '(com|ca|co\\.uk|fr|ae|at|de|it|sg|co\\.jp|jp|' +
+      'cn|es|in|nl|com\\.br|com\\.mx|com\\.au|com\\.tr)' +
+      '|amzn\\.com)',
```

Comment:
This improves readability by a lot! :+1: I would even suggest to further split by independent part, to use parenthesis for indent, and to sort domain names from the left most part (but that one should probably be a separate commit).
```suggestion
      '^(https?://)?' +
      '(((?!music)[^/])+\.)?' +
      '(amazon\\.(' + (
        'ae|at|com\\.au|com\\.br|ca|cn|com|de|es|fr|in' +
        '|it|jp|co\\.jp|com\\.mx|nl|sg|com\\.tr|co\\.uk'
      ) + ')|amzn\\.com)',
```